### ISSUE TITLE
Withdraw request by TA coordinator/institutional moderator

### ISSUE THREAD

**Issue by mmarschler** [DiSSCo/ELViS#189]
#### Description
I (TA coordinator/institutional moderator) was able to withdraw a submitted request although - according to the ELViS manual –  this can only be done by the ELViS admin (p. 20: "And as you can see in the screenshot above, requests can also be “Withdrawn”. This is something only an ELViS admin can do, of course only if a requester asks for this.").

#### Steps to reproduce the issue

1.  log in as TA coordinator/institutional moderator
2. go to requests
3. withdraw request

**Comment by cannedit** [comment]
We will look into this.

**Comment by mmarschler** [comment]
tested - i cannot withdraw a submitted request anymore.

**Comment by mmarschler** [comment]
@cannedit 
I'm totally confused: 
The old ELViS manual said that only Admins can withdraw a submitted request Thats why I started this issue, because I (as TA coordinator) was able to withdraw. This has been changed, I cannot withdraw a submitted request anymore.
BUT: I just saw that the NEW ELViS manual says: "After a request has been approved **a TA coordinator can** – in case that might be necessary for some reason – **withdraw it**, or download the content of the request as a PDF via the buttons in the column “Actions” (page 49). That is exactly this issue that has been changed...
So, shall TA coordinators be able to withdraw requests or not?
